recipes/concoct/meta.yaml

```diff
@@ -20,21 +20,21 @@ requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - python >=3.5
+    - python >=3.8,<=3.10
     - setuptools
     - cython >=0.28.5
     - numpy >=1.8.0
     - gsl
   run:
-    - python >=3.5
+    - python >=3.8,<=3.10
     - setuptools
     - cython >=0.28.5
     - {{ pin_compatible('numpy') }}
     - scipy >=0.13.3
     - pandas >=0.11.0
     - biopython >=1.62b
     - pytz >=2013.9
-    - scikit-learn >=1.1.3
+    - scikit-learn ==1.1.3
```

Comment:
I think it just needs to be pinned to 1.1.* rather than strictly 1.1.3. It  was 1.2 that added the breaking change (if I understand the various threads abotu this)

Reply:
EDIT: I might be wrong there, indeed 1.1.3 was the last point release there... but it appears to work for others. Maybe there is just an ARM specific issue here?

Reply:
EDIT: Sorry might be wrong here, 1.1.3 was the last 1.1 patch indeed